lib/tasks/reporting.rake

```diff
@@ -12,4 +12,17 @@ namespace :reporting do
   task published_attachments_report: :environment do
     Reports::PublishedAttachmentsReport.new.report
   end
+
+  desc "Prints a list of content IDs that documents whose live edition contains a given regular expression"
+  task :matching_docs, [:regex] => :environment do |_, args|
+    regex = Regexp.new(/#{args[:regex]}/)
+
+    Document.where.not(live_edition_id: nil).find_in_batches(batch_size: 1000) do |batch|
```

Comment:
I saw on the Trello card that you were investigating other table,  are there none?

Reply:
I've constrained this PR to just the editionable documents.  I was going to raise in today's stand-up what the scope of the spike is, as I've already hit the 2 days on the ticket.